Makefile

```diff
@@ -1052,6 +1052,7 @@ provision-install: python-install
 	$(INSTALL) -m 0755 setup/openchange_migrate $(DESTDIR)$(sbindir)/
 	$(INSTALL) -m 0755 setup/openchange_newuser $(DESTDIR)$(sbindir)/
 	$(INSTALL) -m 0755 setup/openchange_neworganization $(DESTDIR)$(sbindir)/
+	$(INSTALL) -m 0755 setup/openchange_group $(DESTDIR)$(sbindir)/
```

Comment:
This change yes. But you have to add a change in the packaging to include this file and the packaging code is only available in `trusty` branch in Zentyal/openchange repository.